vcsi/vcsi.py

```diff
@@ -948,6 +948,7 @@ def mxn_type(string):
     try:
         split = string.split("x")
         m = int(split[0])
+        assert (m > 0)
```

Comment:
1. the `assert` is caught by the try/except. We could specifically test and then raise the `argparse.ArgumentTypeError`. Or specify which exceptions are caught: `IndexError, ValueError, AssertionError`
2. My use case does not care about the `n`. As in I am okay with `-g 2x-1` or `-g 2x0`. This works better as a `-c/--columns` if that makes more sense.